skimage/feature/tests/test_haar.py

```diff
@@ -88,6 +88,7 @@ def test_haar_like_feature_precomputed(feature_type):
     img = np.ones((5, 5), dtype=np.int8)
     img_ii = integral_image(img)
     if isinstance(feature_type, list):
+        feature_type = list(feature_type)
```

Comment:
Nitpick: Make `feature_type` a tuple from the start in `parametrize` instead?

Reply:
The extra `list` is used to take a copy of feature_type, which is being mutated with `shuffle` afterwards